pom.xml

```diff
@@ -550,6 +550,17 @@
       <scope>test</scope>
     </dependency>
 
+    <dependency>
+      <groupId>org.checkerframework</groupId>
+      <artifactId>checker-qual</artifactId>
+      <version>2.8.2</version>
```

Comment:
Should be updated to 2.9.0

Reply:
The checker does not run when I change the version to 2.9.0

Reply:
Can you be more precise about what goes wrong? There's no reason that this shouldn't work on 2.9.0, and if it doesn't then that's something I would like to debug.

Reply:
I might have made some mistake when I tried, it does work.